Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
+
+RUN yum install -y -q epel-release && \
+    yum install -y -q python-pip && \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    pip install https://github.com/openmicroscopy/zeroc-ice-py-centos7/releases/download/0.0.3/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+
+RUN pip install omego && \
+    mkdir -p /opt/omero && \
+    cd /opt/omero && \
+    omego download python --sym OMERO.py && \
+    echo /opt/omero/OMERO.py/lib/python > /usr/lib/python2.7/site-packages/omero.pth
+
+RUN pip install https://github.com/manics/omero-prometheus-tools/archive/pip-package.zip
```

Comment:
This seems odd. Shouldn't this be building the source in the current tree.

Reply:
Could do